user-impl/src/bundle/bundle.conf

```diff
@@ -0,0 +1,28 @@
+version = "1"
+name = "user"
+compatibilityVersion = "1"
+system = "user"
+systemVersion = "1"
+nrOfCpus = 0.1
+memory = 268435456
+diskSpace = 200000000
+roles = ["web"]
+components = {
+  user = {
+    description = "user"
+    file-system-type = "universal"
+    start-command = ["user/bin/user"]
+    endpoints = {
+      "user" = {
+        bind-protocol = "http"
+        bind-port = 0
+        services = ["http://:9000/user","http://:9000/api/user?preservePath"]
```

Comment:
After reviewing all file changes in the PR I'm not sure what's so special about `user-service-impl` that it's the only project with a hardcoded `bundle.conf`. 
I'm not familiar with ConductR's support for maven but if it doesn't exist then we need `bundle.conf` in all projects, and if it does, there's probably something special un `user-service-impl`.